tensorrt_llm/_torch/pyexecutor/sampler.py

```
@@ -1123,6 +1124,660 @@ def get_cache_indirection(self) -> torch.Tensor | None:
+            store.past_tokens_cuda[:, seq_slots] = full_tokens
+            # Return the result
+            word_len_dim = 2
+            num_words_dim = 1
+            return torch.any(matches_after_mask.all(dim=word_len_dim), dim=num_words_dim)
+
+        @nvtx_range("_are_stop_words_single_token")
+        def _are_stop_words_single_token(
+            self,
+            seq_slots: torch.Tensor,
+            tokens: torch.Tensor,
+            num_accepted_tokens: torch.Tensor,
```

Ruff (0.15.2)

[warning] Unused method argument: `num_accepted_tokens`

(ARG002)